apps/backend/src/modules/users/user.entity.ts

```diff
@@ -67,6 +75,7 @@ class UserEntity implements Entity {
 		passwordSalt: string;
 	}): UserEntity {
 		return new UserEntity({
+			createdAt: "",
```

Comment:
Not sure, but maybe it's better to bring back `createdAt` initializing, as it was in constructor:
```suggestion
			createdAt: new Date().toISOString(),
```
so when we call this method, this field won't be empty string, but contain real created at date

Reply:
I don't think it needs to be done this way because there is a function in the abstract.model.ts file that adds build time when adding an object to the database

Reply:
Oh, then it's okay as it is